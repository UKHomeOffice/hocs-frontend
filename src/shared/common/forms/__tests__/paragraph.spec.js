import React from 'react';
import Paragraph from '../paragraph.jsx';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';

describe('Paragraph component', () => {

    it('should render with default props', () => {
        expect(
            render(<Paragraph />)
        ).toMatchSnapshot();
    });

    it('should render with content when passed', () => {
        expect(
            render(<Paragraph>Testing the Paragraph component</Paragraph>)
        ).toMatchSnapshot();
    });

});
