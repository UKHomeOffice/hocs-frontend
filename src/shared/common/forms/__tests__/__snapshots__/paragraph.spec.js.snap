// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph component should render with content when passed 1`] = `
<p
  class="govuk-body"
>
  Testing the Paragraph component
</p>
`;

exports[`Paragraph component should render with default props 1`] = `
<p
  class="govuk-body"
/>
`;
